src/databricks/labs/ucx/cli.py

```diff
@@ -471,6 +472,14 @@ def revert_dbsql_dashboards(w: WorkspaceClient, dashboard_id: str | None = None)
     ctx.redash.revert_dashboards(dashboard_id)
 
 
+@ucx.command()
+def join_collection(a: AccountClient, w: WorkspaceClient, workspace_id: int):
```

Comment:
is this an account-level command or workspace-level command? account-level command gets `a: AccountClient` injected, workspace-level command gets `w: WorkspaceClient` injected

Reply:
made it account level